cogs/annual_handover_and_reset.py

```diff
@@ -260,7 +259,7 @@ async def increment_year_channels(self, ctx: TeXBotApplicationContext) -> None:
 
         if final_year_channel:
             await initial_message.edit(
-                content=":hourglass: Archiving \"final-years\" channel... :hourglass:",
+                content=':hourglass: Archiving "final-years" channel... :hourglass:',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes